Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -210,12 +211,29 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
             ScaleEdges();
             ScaleAreas();
 
+            const auto &period = Geom(lev).periodicity();
```

Comment:
Seeing as all the logic in this `#ifdef AMREX_USE_EB` block matches exactly the block in `WarpXInitData.cpp` it is probably worth consolidating all this logic into a function that can be called by both routines instead of duplicating the code. But I think that should go in a different upstream PR.